fail2ban/tests/servertestcase.py

```diff
@@ -1021,7 +1021,11 @@ def testCheckStockJailActions(self):
 						cmd[2] = 'polling'
 					# change log path to test log of jail (to prevent "Permission denied" on /var/logs/ for test-user):
 					elif len(cmd) > 3 and cmd[0] == 'set' and cmd[2] == 'addlogpath':
-						cmd[3] = os.path.join(TEST_FILES_DIR, 'logs', cmd[1])
+						fn = os.path.join(TEST_FILES_DIR, 'logs', cmd[1])
+						# fallback to testcase01 if jail has not an own test log-file (currently should be no matter):
+						if not os.path.exists(fn): # pragma: no cover
+							fn = os.path.join(TEST_FILES_DIR, 'testcase01.log')
+						cmd[3] = fn
```

Comment:
I think that just assumptions in the test were somewhat "wrong" --  We do not have sample log files per jail -- we have them for each filter and there is a test (fail2ban.tests.samplestestcase.FilterSamplesRegex) to verify that we do.  So if you are to replace log file with something appropriate, it should ideally match the filter name.  For majority of jails, jail name matches filter name, so I guess it kinda worked.  Without this "fix" I ran into the same issue on a clean clone as well (so nothing adhockish about my setup), that is why I was puzzled why failure didn't trigger on travis.
But in the big realm of things for this test it doesn't care which file we feed here to the server, as long as it exists.  So, could as well all be testcase01.log, and a little comment that it shouldn't really matter if server doesn't get confused too much before getting killed ;)
So, I will merge and then will push a little DOC/PEP8ing enhancement PR